Automattic/portkey-android
# Updated publish button text to next

## Builds on top of #410 

Fixes #406

## Solution
Updated the `Next Button`'s string resource to `Next`

## Testing
1. Create a new post. 
2. Notice the publish button says Next instead of Publish like below. 

<kbd><img src="https://user-images.githubusercontent.com/1509205/86201114-3758ba80-bb24-11ea-9366-348540b67894.png" width="320"></kbd>

## Reviewing 
Only 1 reviewer is needed but anyone can review. 

## Submitter Checklist
